### ISSUE TITLE
Axis labels plot_age_precision()

### ISSUE THREAD

**Issue by robynforrest** [pbs-assess/gfplot#33]
Please could the axis labels of age_precision() be integers incremented by 1. Cheers.
![age-precision](https://user-images.githubusercontent.com/2694518/40456173-d1edab48-5ea4-11e8-8493-df3bb4ca28cb.png)

**Comment by seananderson** [comment]
It would be tricky to specify axis ticks that would make sense across all species, but because the output is a ggplot object, it's easy to modify after:

```r
g <- get_age_precision(222) %>%
  tidy_age_precision() %>%
  plot_age_precision()

g + scale_x_continuous(breaks = seq(0, 10, 1)) + 
  scale_y_continuous(breaks = seq(0, 10, 1))
```

![image](https://user-images.githubusercontent.com/19349/40499626-5c5058fc-5f37-11e8-8d3e-4e0105e4b645.png)

Note that I also just fixed the jittering in `plot_age_precision()` in the latest version I just pushed. I noticed when looking at your plot that the confidence intervals and dots weren't getting jittered by the same amount. I also changed the precision label to 'secondary age' based on feedback from the synopsis report.
